content/02.intro.md

```diff
@@ -24,4 +24,4 @@ Although deep learning is a distinct subfield of machine learning, it is still a
 It is subject to the many limitations inherent to machine learning, and many best practices for machine learning also apply to deep learning.
 In addition, as with all computational methods, deep learning should be applied in a systematic manner that is reproducible and rigorously tested.
 Ultimately, the tips we collate range from high-level guidance to best practices for implementation, and it is our hope that they will provide actionable, deep learning-specific instruction for both new and experienced deep learning practitioners.
-By making deep learning more accessible for use in biological research, we aim to improve the overall usage and reporting quality of deep learning in the literature, and to enable increasing numbers of researchers to utilize these state-of-the art techniques effectively and accurately.
\ No newline at end of file
+By making deep learning more accessible for use in biological research, we aim to improve the overall usage and reporting quality of deep learning in the literature, and to enable increasing numbers of researchers to effectively and accurately utilize these state-of-the art techniques.
```

Comment:
This creates a split infinitive

Reply:
I tend to ascribe to the Star Trek approach ("to boldly go"), and like split infinitives sometimes :)

But happy to change it!

Thanks for the other fix...I tried to "commit it", but my apologies if I did that wrong and created a merge conflict.